mm2src/mm2_main/tests/docker_tests_main.rs

```diff
@@ -50,15 +50,18 @@ pub fn docker_tests_runner(tests: &[&TestDescAndFn]) {
         pull_docker_image(UTXO_ASSET_DOCKER_IMAGE_WITH_TAG);
         pull_docker_image(QTUM_REGTEST_DOCKER_IMAGE_WITH_TAG);
         pull_docker_image(GETH_DOCKER_IMAGE_WITH_TAG);
+        pull_docker_image(SIA_DOCKER_IMAGE_WITH_TAG);
         remove_docker_containers(UTXO_ASSET_DOCKER_IMAGE_WITH_TAG);
         remove_docker_containers(QTUM_REGTEST_DOCKER_IMAGE_WITH_TAG);
         remove_docker_containers(GETH_DOCKER_IMAGE_WITH_TAG);
+        remove_docker_containers(SIA_DOCKER_IMAGE_WITH_TAG);
 
         let utxo_node = utxo_asset_docker_node(&docker, "MYCOIN", 7000);
         let utxo_node1 = utxo_asset_docker_node(&docker, "MYCOIN1", 8000);
         let qtum_node = qtum_docker_node(&docker, 9000);
         let for_slp_node = utxo_asset_docker_node(&docker, "FORSLP", 10000);
         let geth_node = geth_docker_node(&docker, "ETH", 8545);
+        let sia_node = sia_docker_node(&docker, "SIA", 9980);
```

Comment:
One last question before merging, don't we need to also add a feature flag for this to not run sia docker node in CI tests and avoid the overhead? As I can see, there are no tests that currently use this node.
Related: https://github.com/KomodoPlatform/komodo-defi-framework/pull/2086#discussion_r1572851350

Reply:
Let me remove this line altogether. I have tests locally that I've been using with this, but I want to restructure them before pushing.